examples/support-vector-machine/script.jl

```diff
@@ -1,26 +1,23 @@
 using KernelFunctions
-using MLDataUtils
-using Zygote
-using Flux
 using Distributions, LinearAlgebra
 using Plots
 
-N = 100 #Number of samples
-μ = randn(2, 2) # Random Centers
-xgrid = range(-3, 3; length=100) # Create a grid
-Xgrid = hcat(collect.(Iterators.product(xgrid, xgrid))...)' #Combine into a 2D grid
-y = rand([-1, 1], N) # Select randomly between the two classes
+N = 100  # Number of samples
+μ = randn(2, 2)  # Random Centers
+xgrid = range(-3, 3; length=100)  # Create a grid
+Xgrid = hcat(collect.(Iterators.product(xgrid, xgrid))...)'  # Combine into a 2D grid
+y = rand([-1, 1], N)  # Select randomly between the two classes
 X = zeros(N, 2)
-X[y .== 1, :] = rand(MvNormal(μ[:, 1], I), count(y .== 1))' #Attribute samples from class 1
-X[y .== -1, :] = rand(MvNormal(μ[:, 2], I), count(y .== -1))' # Attribute samples from class 2
+X[y .== 1, :] = rand(MvNormal(μ[:, 1], I), count(y .== 1))'  # Attribute samples from class 1
+X[y .== -1, :] = rand(MvNormal(μ[:, 2], I), count(y .== -1))'  # Attribute samples from class 2
 
-k = SqExponentialKernel(2.0) # Create kernel function
+k = SqExponentialKernel()  # Create kernel function
```

Comment:
```suggestion
k = compose(SqExponentialKernel(), ScaleTransform(2.0))  # Create kernel function
```
This would be the correct equivalent